Automattic/wp-calypso
# Hosting dashboard: clean up Deployments tab

Part of:

- https://github.com/Automattic/dotcom-forge/issues/7924

## Proposed Changes

This PR refactors the code for the hosting dashboard's Deployments tab, by moving them under `client/hosting/deployments/` directory.

## Why are these changes being made?

Consistency and to avoid future confusion.

## Testing Instructions

1. Go to `/sites`.
2. Select an Atomic site.
3. Click the Deployments tab.
4. Smoke test and verify that the feature still works.


## Pre-merge Checklist
